pkg/systemd/dbus.go

```diff
@@ -0,0 +1,47 @@
+package systemd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/containers/libpod/pkg/rootless"
+	"github.com/coreos/go-systemd/v22/dbus"
+	godbus "github.com/godbus/dbus/v5"
```

Comment:
On second reading I realize this is "go-dbus".  But on the first pass through I read "god-bus" and given corona, it made me question things!

Reply:
:rofl: